contracts/UnionZap.sol

```diff
@@ -49,22 +51,33 @@ contract UnionZap is Ownable, UnionBase {
         bytes32[] merkleProof;
```

Comment:
The `claimParam` struct isn't used anywhere and can be removed. Seems like the struct of the same name is available from `IMultiMerkleStash`

Reply:
Fixed in bb7fde01fad50060123aa9805ca55ae10eb92c5b